scripts/sanity_chk/sanitylib.py

```diff
@@ -2219,12 +2219,16 @@ def report_out(self):
             if instance.status in ["error", "failed", "timeout"]:
                 self.log_info_file(self.inline_logs)
         else:
+            try:
+                completed_perc = int((float(self.suite.total_done) / self.suite.total_to_do) * 100)
+            except ZeroDivisionError:
+                completed_perc = 0
```

Comment:
Or:

    completed_perc = 0
    if self.suite.total_to_do > 0:
        completed_perc = int((float(self.suite.total_done) / self.suite.total_to_do) * 100)

which seems more natural, but I don't know how much Python folks prefer try-and-recover over check-and-avoid.

Reply:
agree the above looks more right.

Reply:
Thank you for the comments. I changed the code according to your suggestion.